README.md

```diff
@@ -1,72 +1,96 @@
+
+The following Open Neuroscience talk provides a good high-level introduction to Caiman:    
+https://www.youtube.com/watch?v=5APzPRbzUIA
 
-If you want to use GPU functionality and have a GPU where you're running CaImAn (most likely a Linux system), you'll want, after you build your conda environment, to switch to a GPU build of the tensorflow package (conda list will tell you, after the version string, what build variant you have - you most likely will get an mlk build, but a "conda search -c conda-forge tensorflow" will probably show you some gpu variants you can switch to - pick one appropriate for your conda version, ideally of the same version of tensorflow you otherwise got). If you need help switching versions, reach out to us on the gitter channel.
+The following Nemonic workshops are more in depth:
 
-## Getting Started
+* https://www.youtube.com/watch?v=KjHrjhvhRy0
+* https://www.youtube.com/watch?v=rUwIqU6gVvw
+* https://www.youtube.com/watch?v=NZZ6_zo0YIM
+* https://www.youtube.com/watch?v=z6TlH28MLRo
 
-If you used caimanmanager to unpack the demos and data files, you will find in the caiman\_data folder a set of demos and jupyter notebooks. demo\_pipeline.py and demo\_behavior.py (or their notebook equivalents) are good introductions to the code.
 
-## Papers and data
+# Related repositories :pushpin:
+There are many repositories that make heavy use of Caiman, or will help make using Caiman easier. If you would like your software to be in this list, please ask at Gitter or open an issue. 
 
-### Main paper
-A paper explaining most of the implementation details and benchmarking can be found [here](https://elifesciences.org/articles/38173).
+* [use\_cases repo](https://github.com/flatironinstitute/caiman_use_cases):  additional code (unmaintained) demonstrating how to reproduce results in some Caiman-related papers, and how to use/extend Caiman.
+* [jnormcorre](https://github.com/apasarkar/jnormcorre): [JAX](https://github.com/google/jax) implementation of NoRMCorre for motion correction using JAX acceleration
+* [funimag](https://github.com/paninski-lab/funimag): matrix decomposition for denoising and compression
+* [mesmerize-core](https://github.com/nel-lab/mesmerize-core): parameter optimization, data organization and visualizations with Caiman
+* [improv](https://github.com/project-improv/improv):  a platform for creating online analysis workflows that lets you use Caiman in real time (e.g., for all-optical experiments)
 
+If you have questions about these related packages please reach out to their maintainers directly. 
+
+# Caiman paper
+If you use Caiman and end up publishing, we kindly ask that you [cite Giovannucci et al., 2019](https://elifesciences.org/articles/38173):
 ```
 @article{giovannucci2019caiman,
-  title={CaImAn: An open source tool for scalable Calcium Imaging data Analysis},
+  title={Caiman: An open source tool for scalable Calcium Imaging data Analysis},
```

Comment:
I think this is one of the few name bits we really can't change, because the name of the paper has this capitalisation and the citation should follow it.

Reply:
Good eye. I did replace all, which also messed up a link.